### PR TITLE
Adding MoreInformation resource

### PR DESCRIPTION
**Pull request by filipeotero**
### Purpose

This PR creates a resource for MoreInformation button in "Graph contains outdated Python engine" message.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang 

### FYIs

@RobertGlobant20 @jesusalvino

### CODE REVIEW

**Comment by QilongTang** [comment]
Hi @filipeotero Would you resolve the conflicts with master? I think the recent SDK changes by @pinzart got merged to master first so we need to address local conflicts

File: src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj [hunk 1]
```diff
@@ -1,121 +1,121 @@
...
-    </EmbeddedResource>
-  </ItemGroup>
-  <Target Name="AfterBuildOps" AfterTargets="Build">
-    <ItemGroup>
-      <ExtensionDefinition Include="PythonMigration_ViewExtensionDefinition.xml" />
-      <PythonFixers Include="$(SolutionDir)PythonMigrationViewExtension\MigrationAssistant\python_migration_fixers\*" />
-    </ItemGroup>
-    <Copy SourceFiles="@(ExtensionDefinition)" DestinationFolder="$(OutputPath)\viewExtensions\" />
-    <Copy SourceFiles="@(PythonFixers)" DestinationFolder="$(OutputPath)python_migration_fixers\" />
-  </Target>
```
Comment (QilongTang) [inline src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj:120]: The tab size in this file does not look correct.. Do we need to touch this file? If not, lets revert these changes to follow the original tab spacing

**Review by QilongTang — COMMENTED** [review]
One question

File: src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj (hunk 1 above)
Comment (filipeotero) [reply src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj:120]: Sorry, it had some indentation differences. I will revert it properly. Thanks!

Comment (QilongTang) [reply src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj:120]: Thanks

**Comment by QilongTang** [comment]
Hi @filipeotero The format of that project file still does look correct

**Comment by filipeotero** [comment]
> Hi @filipeotero The format of that project file still does look correct

Now it's done 😅

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
Waiting for PR checks
